backend/src/main/kotlin/org/loculus/backend/service/submission/SubmissionDatabaseService.kt

```diff
@@ -984,18 +987,34 @@ class SubmissionDatabaseService(
             Clock.System.now().toEpochMilliseconds() - timeToStaleInSeconds * 1000,
         ).toLocalDateTime(TimeZone.UTC)
 
-        val numberDeleted = SequenceEntriesPreprocessedDataTable.deleteWhere {
-            statusIs(IN_PROCESSING) and startedProcessingAtColumn.less(staleDateTime)
+        transaction {
```

Comment:
Why do we need this transaction here? The whole service is `@Transactional` already.

Reply:
Thanks, didn't know this!

Reply:
@anna-parker